admin/includes/init_includes/init_errors.php

```diff
@@ -69,7 +69,7 @@
  */
 if (!defined('WARN_DATABASE_VERSION_PROBLEM')) define('WARN_DATABASE_VERSION_PROBLEM','true');
 // check if the installer directory exists, and warn of its existence
-if (WARN_INSTALL_EXISTENCE == 'true') {

```

Comment:
Noooooooo!  This is not needed and will be a pain for people who are working on the next version out of Github.

Reply:
i don't follow why it would be a pain.  can you elaborate your thoughts, please.

Reply:
If you are working in the zen-cart.com Github repo you need both of these flags on.

Reply:
@scottcwilson I'm not sure I understand.

@proseLA I'm undecided. Is this just to avoid defining "2" constants in dev?
Also, is using `false` boolean vs `'true'` string a good idea?

Reply:
i'm a fan of the boolean.  and i'm not opposed to changing it as well as the define that you mentioned above.  the point is to avoid the constants, yes.  i'm not sure why these 2 need to be separated.  IMO, there are so many constants that one needs to figure out which constant to change prior to actually changing it.  when one is doing ZC 24/7; perhaps one remembers all of these things.  but it would be nice to combine so much of that.  especially when some of us look away from ZC and then come back.
@scottcwilson this PR (except for the points raised by @drbyte) should not affect turning both of these flags on in development or elsewhere.  unless i am missing something...

Reply:
> perhaps one remembers all of these things

nah, I just copy in my override files and they take care of it. No thinking or remembering involved! ☺️